frontend/ui/src/pages/Inbox/ConversationsFilter/index.tsx

```diff
@@ -108,20 +111,21 @@ const ConversationsFilter = (props: ConversationsFilterProps) => {
     const {conversationsPaginationData, filteredPaginationData} = props;
     const formatter = new Intl.NumberFormat('en-US');
 
-    if (filteredPaginationData.loaded) {
+    if (
+      filteredPaginationData.filteredTotal != undefined &&
+      filteredPaginationData.filteredTotal != filteredPaginationData.total
+    ) {
       return (
         <div className={styles.filterCount}>
-          {`Filtered: ${filteredPaginationData.filteredTotal} Total: ${filteredPaginationData.total}`}
+          {`Filtered: ${filteredPaginationData.filteredTotal} Total: ${props.conversations.length}`}
         </div>
       );
     }
 
     if (conversationsPaginationData.total) {
       return (
         <div className={styles.filterCount}>
-          {`${formatter.format(
-            filteredPaginationData.filteredTotal || conversationsPaginationData.total
-          )} Conversations`}
+          {`${formatter.format(filteredPaginationData.filteredTotal || props.conversations.length)} Conversations`}
         </div>
```

Comment:
isn't `filteredPaginationData.filteredTotal` only supposed to be displayed in the first check?